user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_including_all_custom_properties.md

```diff
@@ -4,35 +4,22 @@ uid: SLNetClientTest_including_all_custom_properties
 
 # Configuring server-side search to include all custom properties
 
-From DataMiner 10.2.12/10.3.0 onwards, in the *MaintenanceSettings.xml* file, you can configure the DataMiner Cube server-side search engine by specifying one or more search options in the *SLNet.SearchOptions* element.
+From DataMiner 10.2.12/10.3.0 onwards, an option is available through which you can configure the DataMiner Cube server-side search engine by specifying one or more search options in the *SLNet.SearchOptions* element.
 
-1. Stop DataMiner
+1. [Connect to the DMA using the SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
 
-1. Open the file *C:\Skyline Dataminer\MaintenanceSettings.xml*.
+1. In the *advanced* menu, select *Options > SLNet Options > SearchOptions*.
 
-1. In the SLNet.SearchOptions tag, specify the following option:
-
-   | **Option** | **Description** |
-   |--|--|
-   | includeAllCustomProperties | Server-side search engine will search the values of all custom properties. |
-
-This is an excerpt from *MaintenanceSettings.xml* where indexing has been set to “includeAllCustomProperties”:
-
-   ```xml
-   <SLNet>
-     <SearchOptions>includeAllCustomProperties</SearchOptions>
-   </SLNet>
-   ```
+1. Add *includeAllCustomProperties*.
```

Comment:
Perhaps indicate that multiple SearchOptions are separated by a `;` (or link back to the [SearchOptions](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Setting_the_indexing_options_for_the_server-side_search.html) topic that already explains this.